### PR TITLE
Pr bebop progress

### PR DESCRIPTION
**Pull request by tumbili**

### CODE REVIEW

**Comment by tumbili** [comment]
@LorenzMeier Now that stuff is working I can propose a final tune tomorrow.

File: posix-configs/bebop/px4.config [hunk 1]
```diff
@@ -36,4 +40,4 @@ mavlink stream -u 14556 -s HIGHRES_IMU -r 50
 mavlink stream -u 14556 -s ATTITUDE -r 50
 df_bebop_bus_wrapper start
 mavlink boot_complete
-sdlog2 start -e -b 20 -t
+logger start -b 20 -e -t
```
Comment (LorenzMeier) [inline posix-configs/bebop/px4.config]: Can we use a bigger buffer to not drop stuff?

Comment (tumbili) [inline posix-configs/bebop/px4.config]: @LorenzMeier Sure we can, for once we are not out or RAM ;-) I've checked from dropouts and haven't seen any (logger status) but we can still increase it a bit.

**Comment by LorenzMeier** [comment]
Can you also update the Driver framework submodule including the fix? I just rebased that to master, so its good to go.

**Comment by tumbili** [comment]
@LorenzMeier Yes, how large should we make the logger buffer, 40kB?

**Comment by tumbili** [comment]
@LorenzMeier Ok, I just realized that Snapdragon has 200kB so I'm going for the same. I guess those micro-controllers have made me too cautious.

**Pull request merged**